homeassistant/components/kostal_plenticore/sensor.py

```diff
@@ -69,11 +73,7 @@ async def async_setup_entry(
 
     available_settings_data = await plenticore.client.get_settings()
     settings_data_update_coordinator = SettingDataUpdateCoordinator(
-        hass,
-        _LOGGER,
-        "Settings Data",
-        timedelta(seconds=300),
-        plenticore,
+        hass, _LOGGER, "Settings Data", timedelta(seconds=300), plenticore
```

Comment:
Please don't change the formatting of unrelated code.

Reply:
I just used `black` as required in the PR checklist...

Reply:
Looks like you've removed a comma after `plenticore`. That's why it's reformatting this line.